support-frontend/app/views/main.scala.html

```diff
@@ -144,18 +144,18 @@
     @mainElement.fold(emptyDiv, withContent)
 
     <script>
-      function polyfillLoaded() {
+      function guardianPolyfilled() {
         window.guardian.polyfillScriptLoaded = true;
       }
     </script>
 
-    @if(Random.nextBoolean) {
-      <script src="https://polyfill.io/v3/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries&callback=polyfillLoaded"></script>
-      <script>window.guardian.polyfillVersion = 'v3';</script>
-    } else {
-      <script src="https://polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries&callback=polyfillLoaded"></script>
-      <script>window.guardian.polyfillVersion = 'v2';</script>
-    }
+    <script src="https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js?rum=0&features=es6,es7,es2017,es2018,es2019,default-3.6,HTMLPictureElement,IntersectionObserver,IntersectionObserverEntry,fetch,NodeList.prototype.forEach&flags=gated&callback=guardianPolyfilled&unknown=polyfill&cacheClear=1"></script>
```

Comment:
I've tested whether the polyfills we were previously getting (`Array.prototype.find, fetch, Array.prototype.includes, Number.isInteger, Object.values, Object.entries`) work and they're all there in IE11.